### ISSUE TITLE
Linux (mono): v1.18.0 refuses to run.

### ISSUE THREAD

**Issue by Makitk** [KSP-CKAN/CKAN#1791]
**CKAN Version:** v1.18.0 (v1.16.1 works fine)

**Operating System:** Linux (BunsenLabs (Debian 8), with SteamOS repos for nvidia driver updates)

**The issue you are experiencing:** CKAN v1.18.0 won't even show its window, but crashes instead.

**How to recreate this issue:** Run v1.18.0 through mono.

**CKAN error codes (if applicable):** CKAN window won't appear, so none?

**Comment by Makitk** [comment]
[CKAN_v1_18_0_log_2016-06-22.txt](https://github.com/KSP-CKAN/CKAN/files/328387/CKAN_v1_18_0_log_2016-06-22.txt)

**Comment by politas** [comment]
What's your mono version?

**Comment by Makitk** [comment]
$ mono -V
Mono JIT compiler version 3.2.8 (Debian 3.2.8+dfsg-10)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           __thread
	SIGSEGV:       altstack
	Notifications: epoll
	Architecture:  amd64
	Disabled:      none
	Misc:          softdebug 
	LLVM:          supported, not enabled.
	GC:            sgen

**Comment by dragonfi** [comment]
Also happens for me, with the same mono version.
(I'm on Linux Mint 17, based on Ubuntu 14.04.)

Solution was upgrading to the latest (4.4.0) mono version by following the instructions from http://www.mono-project.com/docs/getting-started/install/linux/#debian-ubuntu-and-derivatives .

**Comment by pjf** [comment]
We removed support for testing against mono 3.2.8 in #1715. I'm not sure exactly when we stopped being compatible, but between v1.16.1 and v1.18.0 obviously, so this is something we should be able to find using `git bisect`. Unfortunately I don't have 3.2.8 on my system any more, otherwise I would have bisected this already.

Mono 3.2.8 is filled with bugs compared to more modern releases, but it's also extremely widespread on Linux, particularly the Debian/Ubuntu/Mint ecosystems, which is a lot of our overall user-base.

I have partial blame to for this. I saw that testing on 3.2.8 had been removed as part of the last release, and didn't say "hey, are we *really* sure we want to do that?".

I'm not around anywhere near as much as I'd like, so I'm not the one to champion a return for 3.2.8 support, but it's certainly thing I think would be nice to have, and definitely in the interests of our users.
